pom.xml

```diff
@@ -221,12 +229,16 @@
                                 <repositoryUrl>file://${project.build.directory}/artifacts</repositoryUrl>
                             </repositoryUrls>
                             <launcherArguments>
+                                <vm-options>
+                                    <value>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:3007</value>
+                                </vm-options>
                                 <frameworkProperties>
                                     <org.osgi.service.http.port>${http.port}</org.osgi.service.http.port>
                                 </frameworkProperties>
                             </launcherArguments>
                             <environmentVariables>
                                 <IT_ENCRYPTION_PASSWORD>random</IT_ENCRYPTION_PASSWORD>
+                                <JAVA_OPS>-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:3007</JAVA_OPS>
```

Comment:
Please make this part of a profile if needed otherwise we may get into port clashes, especially on Jenkins.